tests/profile/tracking/test_tracker_token.py

```diff
@@ -0,0 +1,62 @@
+import os
+import time
+import hashlib
+import re
+
+from kano_profile.paths import tracker_token_file
+from kano_profile.tracker.tracker_token import load_token, TOKEN, \
+    generate_tracker_token
+
+
+TOKEN_REGEX = re.compile(r'[0-9a-f]{32}')
+
+
+def validate_token(token):
+    assert len(token) == 32
+    assert TOKEN_REGEX.match(token)
+
+
+'''
+TODO: Try to do something smarter here as TOKEN is created on import which may
+      be invalidated with the manipulations from the other tests
+'''
+def test_token_var():
+    token = load_token()
+
+    assert token == TOKEN
```

Comment:
Given that `TOKEN = load_token()` on import, this doesn't test much. I get the `TODO` note here, but I'd disable the test and mark it as a TODO altogether.

Reply:
Fair point. The only thing that I would say to counter that is that, in its current state, it does at least ensure that the `TOKEN` variable is being set and exported which stops someone accidentally removing this variable or rewriting it.

Reply:
Fair enough. Hopefully it's something we can change in the future. It's things like that that slow down apps in import statements. Bad practice in general.